adguard/config.json

```diff
@@ -17,7 +17,8 @@
   },
   "ports_description": {
     "53/udp": "DNS server port",
-    "80/tcp": "Web interface (Not required for Ingress)"
+    "80/tcp": "Web interface (Not required for Ingress)",
+    "853/tcp": "DNS over HTTPS (Local)"
```

Comment:
This has no effect. But you know that already, as you've tested it?

Reply:
fair. I thought I had it working by mapping 80 to 853 internally hence this change